kolibri/plugins/coach/assets/src/views/exams-page/rename-exam-modal.vue

```diff
@@ -58,7 +60,12 @@
       },
     },
     data() {
-      return { newExamTitle: this.examTitle };
+      return {
+        newExamTitle: this.examTitle,
+        validateTitle: false,
+        validateForm: false,
```

Comment:
It's still not totally clear to me what role `validateForm` plays. Something to do with always showing validation messages after the form has been submitted at least once, maybe?

The logic behind the data variables might be clearer if you name them based on what they really represent, rather than what functionality they drive.

Then, to me at least, this function becomes a little clearer in meaning:

```javascript
titleIsInvalidText() {
  if (this.titleEdited || this.formSubmitted) {
    // return an error string
  }
  return '';
}
```

Reply:
Good point. `formSubmitted` is more clear. Will update.

Reply:
Will update other variables too.

Reply:
Update variable names.